docs/examples/contours/contour_detection.jl

```diff
@@ -0,0 +1,192 @@
+# ---
+# cover: assets/contour_detection.png
+# title: Contour Detection and Drawing
+# ---
+
+# This demonstration shows how to detect contours on binary images
+# The algorithm used is ["Topological Structural Analysis of Digitized Binary Images
+# by Border Following"](https://www.sciencedirect.com/science/article/pii/0734189X85900167) by Suzuki and Abe (Same as OpenCV).#
+
+# Points are represented using CartesianIndex. Contours are represented as a vector of Points.
+# Direction is a single number between 1 to 8. Steps inside functions are marked as they are in the original paper
+
+using Images, TestImages, FileIO
+
+##              N          NE      E       SE      S       SW        W      NW
```

Comment:
Probably don't need this comment here anymore.

Reply:
I think they still make sense here; explaining the inputs of `clockwise` and `counterclockwise`, so I'll keep them unchanged.